### ISSUE TITLE
Morrowind MGE XE Wine & Proton build

### ISSUE THREAD

**Issue by trymeouteh** [Tk-Glitch/PKGBUILDS#402]
https://bugs.winehq.org/show_bug.cgi?id=47940#c24

Me and another guy got the MGE XE mod almost working in Wine. But to get the MWSE function to fully work in MGE XE it requires a special patch which will not be accepted in wine or wine staging. Please make a special wine and proton build with this patch. This will make it easy for myself and anyone else who wants a special wine and proton for MGE XE mod and not have to jump through hoops to get the special wine build compiled.

**Comment by telans** [comment]
You could submit a PR yourself putting this into community-patches

**Comment by Tk-Glitch** [comment]
I'll gladly add it to the community patches, but you'll have to request a build to lutris as that hack isn't suitable for the general purposes builds I'm releasing here. @tannisroot might be interested to know about that.

If you're on a pacman distro, it's actually pretty straightforward to build your own and the only real issue could be the buildtime, if you got a weak CPU.

I could make a proton-tkg build and attach it here but that would be a one-timer and someone else will have to build further updates. Would that be acceptable to you?

**Comment by tannisroot** [comment]
I'll make a build with this once 4.21 comes out.

**Comment by Tk-Glitch** [comment]
The patch seems to not actually work, unless proven otherwise. I'll keep it around for now, as a test.

**Comment by llde** [comment]
@Tk-Glitch  I'm able to start MWSE with MGE XE correctly with this patch (that BTW is mine ). HOwever to work MGE XE must be loaded and require selected dlls native builtin overrides.
Or using MWSE as standalone (almost nobody does this).
Unluckly builtin dlls are always preferreed on dinput8 and d3d8 so injectors will not work without configurations

**Comment by Tk-Glitch** [comment]
Good. I didn't test it as my Morrowind discs are basically lost. I was only assuming due to the open issue comments. That's also why I kept the patch around as I wanted more feedback before drawing an actual conclusion.
Thanks for rectifying!

**Comment by llde** [comment]
yeah the bug will remain open until a stable MWSE with dbghelp reliance is released or until the bug is fixed correctly.
Also mods that rely on MWSE keypress detection may not work without bBackground Keyboard set to 1 (and this is a problem that affect also OBSE with Oblivion and maybe SKyrim with SKSE).
